single_tech_samples/databricks/Cluster_Deployment/databricks/workspace.template.json

```diff
@@ -0,0 +1,66 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "disablePublicIp": {
+      "type": "bool",
+      "defaultValue": false,
+      "metadata": {
+        "description": "Specifies whether to deploy Azure Databricks workspace with Secure Cluster Connectivity (No Public IP) enabled or not"
```

Comment:
Minor punctuation. missing full stop at the end of the sentence :)

Reply:
fixed